src/components/common/EntityInfo.vue

```diff
@@ -15,26 +15,19 @@
 
         <!-- Header With NR Data -->
         <v-list-item-content id="nr-header" v-show="isEntityType">
-
-          <!-- Subtitle -->
-          <v-list-item-subtitle class="entity-subtitle">
-            <span v-if="isTypeBcomp">Incorporate a BC Benefit Company</span>
-            <span v-if="isTypeCoop">Incorporate a BC Cooperative Association</span>
-          </v-list-item-subtitle>
```

Comment:
Did you confirm with Scott that we no longer need this subtitle in the header?

The reason I think we might want to keep this is that the user doesn't have the company yet... they are still in the process of incorporating.

Anyway I'm just assuming things so please check with Scott. Thanks.

Reply:
Ahh yes, I moved it to match the mock ups.